libp2p/protocols/pubsub/pubsubpeer.nim

```diff
@@ -65,6 +65,9 @@ type
     maxMessageSize: int
     appScore*: float64 # application specific score
     behaviourPenalty*: float64 # the eventual penalty score
+    totalTraffic*: int64
+    invalidTraffic*: int64 # data that was parsed by protobuf but was invalid
+    invalidIgnoredTraffic*: int64 # data that couldn't be parsed by protobuf
```

Comment:
My initial proposal just included "usefulTraffic" (ie sum of Valid `Message`) vs "totalTraffic"
What is the benefit of this version? Seems that we are still exposed to spam on the known protobuf fields (ie I could send 1000 subscriptions or whatever)